lib/src/components/NxTableCell/NxTableCell.tsx

```diff
@@ -58,6 +72,34 @@ const NxTableCell = function NxTableCell(props: Props) {
     maskedSort = <NxFontAwesomeIcon icon={faSort} />;
   }
 
+  const filterContent = onFilterChange && (
+    <NxFilterInput disabled={isFilterDisable}
+                   placeholder={filterPlaceholder || ''}
+                   inputId="myId"
+                   onChange={onFilterChange}
+                   value={filter || ''}/>
+  );
+
+  const [filterDropdownLabel, setFilterDropdownLabel] = useState(initialFilterDropdownLabel || '');
+  const onDropdownLinkClick = (item: string) => {
+    if (onDropdownLinkChange) {
```

Comment:
There is certainly a better way to do this.
What I want it's to have the value of the button clicked (here the country) to be store in a property (here the property is filterDropdown) . With this value I should be able to filter the rows of the table (here this happens in the example page)...
![image](https://user-images.githubusercontent.com/57679720/95596798-1fab9f00-0a4e-11eb-9c17-65e68c0de4a9.png)

I also supposed the dropdown label should be updated to the value chosen
![image](https://user-images.githubusercontent.com/57679720/95596737-0f93bf80-0a4e-11eb-8389-9f97e95652e5.png)...